db/db_local/stop_services.go

```diff
@@ -103,16 +104,16 @@ func StopServices(force bool, invoker constants.Invoker, spinner *spinner.Spinne
 	pluginManagerStopError := plugin_manager.Stop()
 
 	// stop the DB Service
-	stopResult, dbStopError := stopDBService(spinner, force)
+	stopResult, dbStopError := stopDBService(context.Background(), spinner, force)
```

Comment:
comment as to why background ok (or remove)

Reply:
:+1: